homeassistant/components/solarlog/const.py

```diff
@@ -43,110 +43,112 @@ class SolarLogSensorEntityDescription(SensorEntityDescription):
         key="power_ac",
         name="power AC",
         icon="mdi:solar-power",
-        native_unit_of_measurement=POWER_WATT,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        device_class=SensorDeviceClass.POWER,
```

Comment:
I assume this was missed for `power_ac` and `power_dc` in a previous refactor.

Reply:
yes, I think I missed these